core/src/main/scala/com/github/johnynek/paiges/Doc.scala

```diff
@@ -823,4 +838,19 @@ object Doc {
    */
   def stack(ds: Iterable[Doc]): Doc =
     intercalate(line, ds)
+
+  /**
+   * build a table with the strings left aligned and
+   * the Docs starting in the column after the longest string.
+   * The Docs on the right are rendered aligned after the rightSep
+   */
+  def tabulate(leftSep: String, fill: Char, rightSep: String, kv: List[(String, Doc)]): Doc =
```

Comment:
I imagine something like
```
a:    foo
bb:  foo
```
so, adding terminators was my motivation. rightSep is the same thing, except when you want them right justified, think bullets on the other side.

I'll add comments and a simpler version (basically space as the fill character, and empty strings for the left and right separators).